doc/manual/manual.md

```diff
@@ -2416,6 +2414,295 @@ In this case, doing the verification compositionally doesn't save
 computational effort, since the functions are so simple, but it
 illustrates the approach.
 
+### Compositional Verification and Mutable Allocations
```

Comment:
The documentation. Due to the way the SAW manual is set up, I describe compositional verification in both the LLVM and MIR backends here, although this patch doesn't change anything in the way that the LLVM backend works.